app/assets/stylesheets/overrides/sufia.css.scss

```diff
@@ -0,0 +1,22 @@
+#home_header {
```

Comment:
Avoid using id selectors<br>Selector `home_header` should be written in lowercase with hyphens